coc/war_clans.py

```diff
@@ -150,17 +152,21 @@ def defenses(self) -> typing.List["WarAttack"]:
     
     @property
     def average_attack_duration(self) -> int:
-        """:class:`int`: Returns the average duration of all clan member's attacks this war."""
+        """:class:`int`: Returns the average duration of all clan member's
+        attacks this war."""
         count = len(self.attacks)
         total_duration = sum(attack.duration for attack in self.attacks)
         return int(total_duration/count) if count > 0 else 0
 
     def get_member(self, tag: str) -> typing.Optional[ClanWarMember]:
-        """Get a member of the clan for the given tag, or ``None`` if not found.
+        """
+        Get a member of the clan for the given tag, or ``None`` if not found.
 
         Returns
         --------
-        The clan member who matches the tag.: Optional[:class:`ClanWarMember`]"""
+        ClanWarMember or None
+            The clan member who matches the tag.: Optional[:class:`ClanWarMember`]"""
+
         tag = correct_tag(tag)
         if not self._members:
             _ = self.members
```

Comment:
Does this server a purpose?

Reply:
I guess this shall clarify that this returns None if no clan member matches the tag.